core/unix/os.c

```diff
@@ -6039,7 +6035,7 @@ handle_execve(dcontext_t *dcontext)
      */
     char *fname;
     bool x64 = IF_X64_ELSE(true, false);
-    bool expect_to_fail = false;
+    bool expect_to_fail UNUSED = false;
```

Comment:
But it's used on L6151:
```
if (should_inject && DYNAMO_OPTION(early_inject) && !expect_to_fail) {
```


Is unused a no-op if it is actually used?

Reply:
Used only for LINUX, not OSX.  Yes, it's a nop if it's actually used, so doesn't hurt LINUX build.